content/topics/java-specific/string-and-number-manipulation/_index.md

```diff
@@ -1,3 +1,52 @@
 ---
 title: Strings and numbers in Java
----
\ No newline at end of file
+---
+
+In Java there are multiple ways to manipulate both strings and numbers to work in a manner that is required. These methods can play a vital role in the business logic of a system.
+
+For example: Say you have a list of 100 books and you need to provide the functionality to search for a book. Without being able to manipulate a String
+the user would always have to search for the full title of a book and ensure that every letter's case, matches that of the title.
+
+Java has built in methods to assist with just these kinds of situations.
+
+## Manipulating Strings
+- [Converting](https://www.geeksforgeeks.org/different-ways-for-integer-to-string-conversions-in-java/) variables to Strings.
```

Comment:
I am not sure if I would label converting an integer to string as string manipulation. Here is something to look into maybe 
https://www.guru99.com/java-strings.html